packages/block-editor/src/components/use-setting/index.js

```diff
@@ -50,19 +50,19 @@ const deprecatedFlags = {
 };
 
 /**
- * Hook that retrieves the setting for the given editor feature.
+ * Hook that retrieves the editor setting.
```

Comment:
Should we clarify this to state it's not for all settings, but https://github.com/WordPress/gutenberg/pull/31587#issuecomment-838382549 ?